extmod/nimble/modbluetooth_nimble.c

```diff
@@ -44,7 +44,7 @@
 #define MICROPY_PY_BLUETOOTH_DEFAULT_NAME "PYBD"
 #endif
 
-#define DEBUG_EVENT_printf(...) //printf(__VA_ARGS__)
```

Comment:
I don't think this is dead code, I think it is for enabling debug prints.

A common pattern I have seen is something like this:

```C
#define DEBUG_EVENT 0
#if DEBUG_EVENT
#define DEBUG_EVENT_printf(...) printf(__VA_ARGS__)
#else
#define DEBUG_EVENT_printf(...)
#endif
```

They you just have to change 0 to 1 to enable debug printing. Maybe this would be better?

Reply:
Yes that's exactly what I'm saying in the commit message. But same remark: who knows which ones are really worth it?